core/src/main/scala/com/criteo/cuttle/Executor.scala

```diff
@@ -203,6 +210,23 @@ case class Execution[S <: Scheduling](
     hasBeenCancelled
   }
 
+  def forceSuccess()(implicit user: User): Boolean = {
```

Comment:
You don't use a return type so I think it can be Unit

Reply:
Done